src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -10636,33 +10636,31 @@ void Player_InitCommon(Player* this, PlayState* play, FlexSkeletonHeader* skelHe
     Collider_SetQuad(play, &this->shieldQuad, &this->actor, &D_808546A0);
 }
 
-static void (*D_80854738[])(PlayState* play, Player* this) = {
-    /* 0x0 */ func_80846648,
-    /* 0x1 */ func_808467D4, // From time travel
-    /* 0x2 */ func_80846660,
-    /* 0x3 */ func_808468A8,
-    /* 0x4 */ func_808468E8,
-    /* 0x5 */ func_808469BC,
-    /* 0x6 */ func_80846A68,
-    /* 0x7 */ func_80846978,
-    /* 0x8 */ func_8083CA54,
-    /* 0x9 */ func_8083CA54,
-    /* 0xA */ func_8083CA54,
-    /* 0xB */ func_8083CA54,
-    /* 0xC */ func_8083CA54,
-    /* 0xD */ func_8083CA20,
-    /* 0xE */ func_8083CA54,
-    /* 0xF */ func_8083CA9C,
+static void (*sStartModeFuncs[PLAYER_START_MODE_MAX])(PlayState* play, Player* this) = {
+    Player_StartMode0,   // PLAYER_START_MODE_0
+    Player_StartMode1,   // PLAYER_START_MODE_1
+    Player_StartMode2,   // PLAYER_START_MODE_2
+    Player_StartMode3,   // PLAYER_START_MODE_3
+    Player_StartMode4,   // PLAYER_START_MODE_4
+    Player_StartMode5,   // PLAYER_START_MODE_5
+    Player_StartMode6,   // PLAYER_START_MODE_6
+    Player_StartMode7,   // PLAYER_START_MODE_7
+    Player_StartModeUnk, // PLAYER_START_MODE_8
+    Player_StartModeUnk, // PLAYER_START_MODE_9
+    Player_StartModeUnk, // PLAYER_START_MODE_10
+    Player_StartModeUnk, // PLAYER_START_MODE_11
+    Player_StartModeUnk, // PLAYER_START_MODE_12
+    Player_StartMode13,  // PLAYER_START_MODE_13
+    Player_StartModeUnk, // PLAYER_START_MODE_14
```

Comment:
Probably a moot point once these get named, but `Player_StartModeUnk` should be associated with `PLAYER_START_MODE_14`. A quick scan of Link actor spawns suggest that modes 8-12 go unused.

Reply:
I went ahead and just labeled them as unused and changed the func name. I thought they were unused, but didnt feel like checking if that was true until later. Thanks for doing so